docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -20,7 +20,8 @@ spec:
 | Field | Type | Description | Required |
 |-|-|-|-|
 | name | string | The application name. | Yes (if you want to create PipeCD application through the application configuration file) |
-| envName | string | The environment name. You need to make sure that the environment name is unique in your project. | No |
+| envName | string | The environment name. You need to make sure that the environment name is unique in your project. Deprecated. | No |
```

Comment:
```suggestion
| envName | string | The environment name. You need to make sure that the environment name is unique in your project. | No (deprecated) |
```
nits

Reply:
For sure. Let me do that by the separated PR.